packages/web-components/src/components/cbp-toast/cbp-toast.specs.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Toast/Specifications" />
+
+# cbp-toast
+
+## Purpose
+
+* Toast is intended to temporarialy display messages to users such as error or warning messages about an form input, or confrimation of a file upload. It is the intent of the designer that for things
```

Comment:
We need to refine this. You should never use toast for form validation. Confirmation misspelled.